rfc/.template.md

```
@@ -0,0 +1,105 @@
+
+---
+
+## Table of Contents
+
+1. Introduction
+2. Terminology
+3. Protocol Overview
+4. Detailed Specification
+5. Security Considerations
+6. Privacy Considerations
+8. References
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 7; Actual: 8; Style: 1/2/3

(MD029, ol-prefix)